app/car.py

```diff
@@ -0,0 +1,59 @@
+from typing import List, Union
+from app.shop import Shop
+import math
+
+
+def distance_calculation(
+        point_custom: List[Union[int, float]],
+        point_shop: List[Union[int, float]]
+) -> float:
+    return math.hypot(
+        point_shop[0] - point_custom[0],
+        point_shop[1] - point_custom[1]
+    )
+
+
+def get_min_distance_shop(
+        location_customers: list,

```

Comment:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contains string elements has typehint list[str]